src/test/java/com/aws/greengrass/certificatemanager/certificate/CISShadowMonitorTest.java

```diff
@@ -590,4 +666,82 @@ private static ConnectivityInfo connectivityInfoWithRandomHost() {
                     .build();
         }
     }
+
+    /**
+     * A ScheduledThreadPoolExecutor that executes using a provided ExecutorService.
+     */
+    static class DelegatedScheduledExecutorService extends ScheduledThreadPoolExecutor {
```

Comment:
why? Just create a scheduled executor using `Executors.newScheduledExecutor`

Reply:
previous changes removed ses, so was following the spirit of that (https://github.com/aws-greengrass/aws-greengrass-client-device-auth/pull/60),  totally fine either way

Reply:
if you need a ses, then create one the normal way. Doing this is unnecessary and implies that there is a reason why this is needed.

Reply:
👍 will fix

Reply:
done